stdlib/buffer.gr

```diff
@@ -376,6 +376,68 @@ provide let addString = (string, buffer) => {
   buffer.len += bytelen
 }
 
+/**
+ * Gets the UTF-8 encoded character at the given byte index.
```

Comment:
If the operation is intended to get a character starting at a byte index then if you point your index in the middle of a UTF-8 character then what would the expectation be?

Reply:
I'm pretty sure it would either return a character in the case that the rest was a valid char (I can't recall if thats ever the case), but more likely `MalformedUnicode`.